crates/runtime/src/env/traits/address_locator.rs

```diff
@@ -0,0 +1,8 @@
+/// Computes the address of an item.
+///
+/// The algorithm must be deterministic.
+pub trait AddressLocator<T> {
```

Comment:
As written above - I don't think that the notion of `Locator` best describes the code.